File: Cappacita/ex_17.js
// 2) Crie as funções parametrizadas a seguir:

// a) Uma função que receba dois valores por parâmetro e multiplique um pelo
// outro

function multiplicacao(valor1, valor2) {
    subtrair = valor1 * valor2;
    console.log(subtrair)    
}

// b) Uma função que receba dois valores e retorne o menor deles, se forem
// iguais retorne que são iguais
function menorValor(num1, num2) {
    if (num1 > num2){
        console.log("Valor " + num2 + " e menor")
    }else if (num1 < num2){
        console.log("Valor " + num1 + " e menor")
    }else{
        console.log("O valores são iguais")
    }
    
}
// c) Uma função que receba um array e imprima todos os itens na tela

var listaDadosClientes = [
    {
    nome: 'Daiane',
    idade: 24,
    endereco: 'Rua Freire, 24, Consolação, São Paulo'
    },
    {
    nome: 'Gabriel',
    idade: 19,
    endereco: 'Rua Armando sá, 47, Perdizes, São Paulo'
    },
    {
    nome: 'Bruno',
    idade: 37,
    endereco: 'Av. Santos Dumont, 978, Belo Horizonte, Minas Gerais'
    }
    ]
    
function lerArray(array) {
    for(cliente in array){
        console.log(array[cliente])
    }
};
// d) Uma função que receba um o array listaDadosCliente e retorne outro array
// contendo somente os endereços destes clientes

function retornaEndereco(array) {
    lista_end = []
    for (end in array){
        lista_end[end] = array[end].endereco
    }
   return lista_end
}
// Uma função que receba um o array listaDadosCliente e retorne outro array
// contendo somente os nomes desses clientes

function retornaNome(array) {
    lista_end = []
    for (end in array){
        lista_end[end] = array[end].nome
    }
   return lista_end
}

multiplicacao(2, 6)
menorValor(2,2)
menorValor(7,8)
menorValor(9,6)
lerArray(listaDadosClientes)
console.log(retornaEndereco(listaDadosClientes))
console.log(retornaNome(listaDadosClientes))

File: Cappacita/ex_17.py
# 2) Crie as funções parametrizadas a seguir:

# a) Uma função que receba dois valores por parâmetro e multiplique um pelo
# outro

def multiplique(valor1, valor2):
    multiplicar =  valor1 * valor2
    print(multiplicar)


# b) Uma função que receba dois valores e retorne o menor deles, se forem
# iguais retorne que são iguais

def menorValor(valor1, valor2):
    if valor1 > valor2:
        print(f"O valor {valor2} e o menor valor entre {valor1} e {valor2}")
    elif valor1 < valor2:
        print(f"O valor {valor1} e o menor valor entre {valor1} e {valor2}")
    else:
        print(f"O valor {valor1} e valor {valor2} são iguais")

# c) Uma função que receba um array e imprima todos os itens na tela

listaDadosClientes = [{
'nome': 'Daiane',
'idade': 24,
'endereco': 'Rua Freire, 24, Consolação, São Paulo'
},
{
'nome': 'Gabriel',
'idade': 19,
'endereco': 'Rua Armando sá, 47, Perdizes, São Paulo'
},
{
'nome': 'Bruno',
'idade': 37,
'endereco': 'Av. Santos Dumont, 978, Belo Horizonte, Minas Gerais'
}]

def lerArray(array):
    for nome in range(len(array)):
        print("O nome do cliente e {}, ele tem {} anos , mora em {}".format(array[nome]['nome'], array[nome]['idade'], array[nome]['endereco'])) 

# d) Uma função que receba um o array listaDadosCliente e retorne outro array
# contendo somente os endereços destes clientes


def novoArray_end(array):
    lista_endecos = []
    for end in range(len(array)):
        enderecos = {}
        enderecos["endereco"] = array[end]['endereco']
        lista_endecos.append(enderecos)
    print(lista_endecos)
    
# Uma função que receba um o array listaDadosCliente e retorne outro array
# contendo somente os nomes desses clientes


def novoArray_nome(array):
    lista_nome = []
    for end in range(len(array)):
        nome = {}
        nome["nome"] = array[end]['nome']
        lista_nome.append(nome)
    print(lista_nome)


multiplique(2, 5)
menorValor(5, 5)
menorValor(6, 7)
menorValor(9, 10)
lerArray(listaDadosClientes)
novoArray_end(listaDadosClientes)
novoArray_nome(listaDadosClientes)